src/ispc.cpp

```diff
@@ -1552,8 +1564,7 @@ std::string Target::GetTripleString() const {
             Error(SourcePos(), "Arm is not supported on Windows.");
             exit(1);
         } else if (m_arch == Arch::aarch64) {
-            Error(SourcePos(), "Aarch64 is not supported on Windows.");
-            exit(1);
+            triple.setArchName("arm64");
```

Comment:
`arm64` is Apple specific name of `aarch64`. So probably it should be `aarch64`, though clang seem to accept equally well `aarch64-none-windows-msvc19.20.0` and `arm64-none-windows-msvc19.20.0` as `--target=<val>` argument.

I would check what is the default target triple on Windows ARM to figure out what is the correct value is.

Reply:
I think the triple we are using is aarch64-pc-windows-msvc

Reply:
This makes sense and that's what I expected, thanks for conforming.

Reply:
Thanks for catching that. I committed changes that address this (and the cmake changes.) Let me know if any other issues need addressed.